lib/elixir_bench/benchmarks/benchmarks.ex

```diff
@@ -63,6 +73,18 @@ defmodule ElixirBench.Benchmarks do
     Repo.all(Job)
   end
 
+  def get_or_create_job(repo, attrs) do
```

Comment:
I like introducing this abstraction! :+1:

Reply:
Speaking of abstraction I like - I think this deserves unit tests. Like the 2 major cases, but also what happens if we get 2 commits with the same sha but from different repositories (see my argument above)

Reply:
yeah, I added test cases to cover this... in the future if we care about the branch name and not just the repo and commit sha the tests will cover this scenario and fail